Fix build with llvm-16:
/usr/ports/pobj/gnustep-libobjc2-1.8.1/libobjc2-1.8.1/objcxx_eh.cc:76:2: error: unknown type name 'unexpected_handler'
        unexpected_handler unexpectedHandler;

https://github.com/gnustep/libobjc2/commit/0ae4e4ceb77410c619bf597d6f0b0842d5f078a6

Index: objcxx_eh.cc
--- objcxx_eh.cc.orig
+++ objcxx_eh.cc
@@ -2,7 +2,6 @@
 #include <stdio.h>
 #include "dwarf_eh.h"
 #include "objcxx_eh.h"
-#include <exception>
 
 extern "C" 
 {
@@ -14,6 +13,14 @@ namespace __cxxabiv1
 }
 
 using __cxxabiv1::__class_type_info;
+
+// Define some C++ ABI types here, rather than including them.  This prevents
+// conflicts with the libstdc++ headers, which expose only a subset of the
+// type_info class (the part required for standards compliance, not the
+// implementation details).
+
+typedef void (*unexpected_handler)();
+typedef void (*terminate_handler)();
 
 namespace std
 {
